devtools/gradle/pom.xml

```diff
@@ -46,14 +46,42 @@
                 </os>
             </activation>
             <properties>
-                <gradle.executable>..\gradlew.bat</gradle.executable>
+                <gradle.executable>gradlew.bat</gradle.executable>
             </properties>
         </profile>
+        <profile>
+            <id>run-gradle-clean</id>
+            <activation>
+                <file>
+                    <missing>${basedir}/src</missing> <!-- only for this pom, so the clean's only run once for all -->
+                </file>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-clean-plugin</artifactId>
```

Comment:
Shouldn't this be 
```xml
<groupId>org.codehaus.mojo</groupId>
<artifactId>exec-maven-plugin</artifactId>
```
instead?
clean plugin doesn't support an `executable`.

Reply:
oops - left-over. removed it.